nativelib/src/main/resources/platform.c

```diff
@@ -0,0 +1,20 @@
+#ifdef _WIN32
+#include <Windows.h>
+#endif
+
+int scalanative_platform_is_windows() {
```

Comment:
@asoltysik Your original code was correct wrt to identifying if something is windows or not.